__tests__/ExpensiMark-HTML-test.js

```diff
@@ -61,8 +61,8 @@ test('Test critical markdown style links', () => {
     + '[link with smart quotes ‘’“”](https://google.com) '
     + '[link with someone@expensify.com email in it](https://google.com)';
     const resultString = 'Testing '
-    + '<a href="http://expensify.com" target="_blank" rel="noreferrer noopener"><del>strikethrough</del> <strong>bold</strong> <em>italic</em></a> '
-    + '<a href="http://expensify.com" target="_blank" rel="noreferrer noopener"><del>strikethrough</del> <strong>bold</strong> <em>italic</em> test.com</a> '
+    + '<a href="https://expensify.com" target="_blank" rel="noreferrer noopener"><del>strikethrough</del> <strong>bold</strong> <em>italic</em></a> '
+    + '<a href="https://expensify.com" target="_blank" rel="noreferrer noopener"><del>strikethrough</del> <strong>bold</strong> <em>italic</em> test.com</a> '
```

Comment:
Why were these tests changed?

Reply:
That's the fail test after I create the PR 😅
[fix Test critical markdown style links test](https://github.com/Expensify/expensify-common/pull/455/commits/804310bd10d8279e5dfc273c5dff62e19894cebd)

Reply:
You're right, `main` branch is failing those tests too.
We proly missed it in this PR https://github.com/Expensify/expensify-common/pull/451

<img width="667" alt="image" src="https://user-images.githubusercontent.com/29673073/174312133-2fd56c4b-de64-4205-8aee-5751fa232319.png">

Reply:
yep

Reply:
Done! I adding a new rule to `htmlToTextRules` to fix the test and fix the message text on the LHN to show correctly.